### PR TITLE
SpriteInfo: Radio Buttons Toggles + Better Classnames

### LINKED ISSUE #156: Clicking any part of hide/show buttons should hide/show sprite
### Expected Behavior
Clicking anywhere inside the border of the button should change the sprite's visibility

### Actual Behavior
User must click the "eye" icon to show/hide the sprite. I assume the same goes for the draggable button

### PR DESCRIPTION
**Pull request by lifeinchords**
Fixes #156 + refactored classnames to style guide conventions

fix-radio-button-toggle mov

### CODE REVIEW

File: src/components/sprite-info/sprite-info.css [hunk 1]
```diff
@@ -31,49 +31,47 @@ $form-radius: calc($space / 2);
...
 }
 
-.icon.is-active { opacity: 1; }
+.radio { opacity: 0.4; cursor: pointer; }
 
-.radio-icon { 
-    opacity: 0.4;
-}
+.radio.is-active   { opacity: 1; }
+.radio.is-disabled { cursor: default; }
```
Comment (paulkaplan) [inline src/components/sprite-info/sprite-info.css]: Just rehashing what we talked about on slack: I think we should keep any CSS that is linked to a javascript action (like `cursor` <=>`onClick`) together on the same element. i.e. if the image has the onClick, then the image should have the cursor property also. 

We also talked about using more semantic markup here (button groups or actual radios) to get this kind of thing for free and also make things more tab/accessibility friendly. But that doesn't have to be part of this PR.

Comment (lifeinchords) [inline src/components/sprite-info/sprite-info.css]: thanks @paulkaplan, I updated this PR with these suggestions. Let's save refactor to input to another issue.

**Comment by paulkaplan** [comment]
@lifeinchords nice LG. Gotta wait to merge until after play test though right?

**Comment by lifeinchords** [comment]
great.. ya, merge later.

**Pull request merged**